src/Transports/MassTransit.AmazonSqsTransport/Configuration/AmazonSqsBusFactoryConfiguratorExtensions.cs

```diff
@@ -38,5 +39,18 @@ public static void LocalstackHost(this IAmazonSqsBusFactoryConfigurator configur
                 h.Config(new AmazonSimpleNotificationServiceConfig { ServiceURL = "http://localhost:4566" });
             });
         }
+
+        public static void UseDefaultHost(this IAmazonSqsBusFactoryConfigurator configurator)
```

Comment:
I would add a nullable argument (same as used by the .Host) for a configure callback on each of these.